tests/test_purge.py

```diff
@@ -131,20 +133,19 @@ def test_scheme_port(client_auth, requests_mocker):
     assert future.result()
 
 
-def test_response_fails(client, requests_mocker, no_retries):
+@responses.activate
+def test_response_fails(client,  no_thread_retries):
     """Requests fail with a FastPurgeError if API gives unsuccessful response."""
-
-    requests_mocker.register_uri(
-        method='POST',
-        url='https://fastpurge.example.com/ccu/v3/delete/cpcode/production',
-        status_code=503,
-        reason='simulated internal error')
-
+    url = 'https://fastpurge.example.com/ccu/v3/delete/cpcode/production'
+    # Decrease backoff, otherwise the test will run for 5 minutes
+    os.environ["FAST_PURGE_RETRY_BACKOFF"] = "0.001"
```

Comment:
This change will not be reverted at the end of the test.  This means for other tests run during the same process, it'll be effectively random whether FAST_PURGE_RETRY_BACKOFF=0.001 or not, based on whether each test function ran before or after this one.

Could you please use monkeypatch.setenv instead so it's automatically reverted?